examples/amphtml-ads/dianomi.a4a.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html ⚡ lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Dianomi Ad</title>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
```

Comment:
You have a couple of lint errors in this file. You can expose them by running `amp validate-html-fixtures` let me know if you have any issue.

Reply:
The errors have now been resolved.